classes/image_types_ostree.bbclass

```diff
@@ -157,21 +157,14 @@ IMAGE_CMD:ostreecommit () {
 IMAGE_TYPEDEP:ostreepush = "ostreecommit"
 do_image_ostreepush[depends] += "aktualizr-native:do_populate_sysroot ca-certificates-native:do_populate_sysroot"
 do_image_ostreepush[lockfiles] += "${OSTREE_REPO}/ostree.lock"
+do_image_ostreepush[network] = "1"
 IMAGE_CMD:ostreepush () {
-    # send a copy of the repo manifest to backend if available
-    local SEND_MANIFEST=""
-    # check if garage-push supports the --repo-manifest option before trying
-    if $(garage-push --help | grep -q '^\s*--repo-manifest') && [ -f ${IMAGE_ROOTFS}${sysconfdir}/manifest.xml ]; then
-        SEND_MANIFEST="--repo-manifest ${IMAGE_ROOTFS}${sysconfdir}/manifest.xml"
```

Comment:
We can add a variable outside the function with the manifest location that can be changed/cleaned to not send the manifest
```
OSTREE_SEND_MANIFEST ?= "${IMAGE_ROOTFS}${sysconfdir}/manifest.xml"

IMAGE_CMD:ostreepush () {
    # send a copy of the repo manifest to backend if available
    local SEND_MANIFEST=""
    if [ -f "${OSTREE_SEND_MANIFEST}" ]; then
        SEND_MANIFEST="--repo-manifest ${OSTREE_SEND_MANIFEST}"
    fi
```